gateway/1.5/modules/ROOT/pages/policies-pdk-prerequisites.adoc

```diff
@@ -1,5 +1,105 @@
+
+The PDK tutorials assume that you have xref:flex-gateway.adoc[Downloaded Flex Gateway] and have some prior knowledge of the product.
+
+
+== Rust Requirements
+
+. Git
++
+To download and install, see the https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Installing Git ^].
+
+. Rust programming language (version 1.65.0 or later) 
++
+To install or update Rust, run the following curl command:
++
+[source,ssh]
+----
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+----
++
+NOTE: If you do not have Curl installed, see https://curl.se/download.html[Install Curl ^].
+
+. Rust WebAssembly System Interface (WASI) Crate
++
+To install or update Rust WASI, run the following command:
++
+[source,ssh]
+----
+rustup target add wasm32-wasi
+----
+
+. Cargo Generate
++
+To install or update Cargo Generate, run the following command:
++
+[source,ssh]
+----
+cargo install cargo-generate
+----
+
+== Supported Operating Systems
+
+* macOS 
+* Ubuntu
++ 
+To use Ubuntu, execute the following commands to install the required packages after installing the <<rust-requirements>>:
++
+[source,ssh]
+----
+sudo apt install -y build-essential
+sudo apt install -y pkg-config
+sudo apt install -y libssl-dev
+----
+* Windows
++
+To execute the `make` commands included in the PDK, install:
++
+. https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.3[PowerShell on Windows ^]
+. Make, with either:
+** https://gnuwin32.sourceforge.net/packages/make.htm[Make for Windows ^]
+** Chocolatey: 
++
+[source,ssh]
+----
+choco install make
+----
+
+[[cli]]
+== Anypoint CLI 
+
+To configure the Anypoint CLI for PDK:
+
+. xref:anypoint-cli::install.adoc[Install Anypoint CLI 4.x].
+. xref:anypoint-cli::auth.doc[Authenticate to the Anypoint Platform CLI] using a xref:access-management::connected-apps-overview.adoc[Connected App].
++
+Authenticate with a Connected App by providing the app's client ID and client secret.
++
+NOTE: No extra permissions are required to download the Anypoint CLI PDK Plugin. However, to complete xref:policies-pdk-publish-policies.adoc[], the Connected App must have the `Exchange Contributor` scope applied.
+
+. Run the following command to install the Anypoint CLI PDK Plugin:
++
+[source,ssh]
+----
+anypoint-cli-v4 plugins:install @mulesoft/anypoint-cli-pdk-plugin
+----
+
+== Docker 
+
+Docker is only required to use the xref:policies-pdk-debug-local.adoc[local testing environment] provided with PDK. You can use all other PDK features without docker.
+
+To install Docker, see https://docs.docker.com/engine/install/[Install Docker Engine ^].
+
+== Visual Studio (Suggested IDE)
+
+You can use PDK in any IDE or text editor. However, MuleSoft suggests that you use https://code.visualstudio.com/[Microsoft Visual Studio ^] with the https://code.visualstudio.com/docs/languages/rust[Rust in Visual Studio Code Plugin ^] installed.
```

Comment:
```suggestion
You can use PDK in any IDE or text editor. However, MuleSoft suggests that you use https://code.visualstudio.com/[Microsoft Visual Studio ^] with the https://code.visualstudio.com/docs/languages/rust[Rust in Visual Studio Code Plugin ^] installed.
```

Reply:
Yeah, that makes sense. It is only really suggested because it provides the Rust plugin and there will be a mulesoft PDK VScode plugin for GA, but I guess we can't mention that in the docs yet. I think your suggestion works for now.